shopinvader_search_engine_image/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_shopinvader_search_engine_image
```

Comment:
```suggestion
from . import test_shopinvader_search_engine_image
from . import test_images
```

Reply:
I still need to cleanup the tests and `test_images` is the previous test file I need to rewrite.